ote/src/cljs/ote/app/controller/admin_transit_changes.cljs

```diff
@@ -93,12 +97,21 @@
                  {:on-success (tuck/send-async! ->SetSingleDetectionServiceId service-id)}))
     app))
 
+(define-event ForceInterfaceImportForGivenServiceSuccess [response]
+  {}
+  (assoc app :flash-message "GTFS paketti ladattu onnistuneesti"))
```

Comment:
Ok, however the current flash-message is easy to miss.
If the result was embedded to the view, next to the button, using e.g. `ote.ui.common/generic-help` or `help` and styled blue like a few views already do, 
it would be persistent on the UI until user navigates away.